docs/intro/install.md

```diff
@@ -5,13 +5,15 @@ This guide shows how to install Krustlet.
 ## From the Binary Releases
 
 Every release of Krustlet provides compiled releases for a variety of Operating Systems. These
-compiled releases can be manually downloaded and installed.
+compiled releases can be manually downloaded and installed. Please note these instructions will work
+on Linux, MacOS, and Windows (in Powershell)
 
 1. Download your desired version from [the releases
    page](https://github.com/deislabs/krustlet/releases)
 1. Unpack it (`tar -xzf krustlet-v0.3.0-linux-amd64.tar.gz`)
 1. Find the desired Krustlet provider in the unpacked directory, and move it to its desired
-   destination (`mv krustlet-wasi /usr/local/bin/`)
+   destination somewhere in your `$PATH` (e.g. `mv krustlet-wasi /usr/local/bin/` on unix-like
+   systems or `mv krustlet-wasi.exe C:\Windows\system32\` on Windows)
```

Comment:
I think it'd be better to place it under `C:\\Windows` rather than `system32\`. system32 should be reserved for system tools, much like how we don't touch `/bin` in unix.

Reply:
I couldn't find a good equivalent to `/usr/local/bin`. `C:\Windows` isn't in the default path (at least on my recent fresh install of windows 10. It did have `%USERPROFILE%\AppData\Local\Microsoft\WindowsApps`, but I am not sure if that exists on every machine. What do you think would be the best option?

Reply:
So here's what I find in my system variables (slightly different than user variables) for PATH: 

![image](https://user-images.githubusercontent.com/1360539/88319749-72887c80-ccd1-11ea-9612-f24815cf2647.png)

`%SystemRoot%` equates to C:\\ on my system:

```
PS C:\Users\me> echo $Env:SystemRoot
C:\Windows
```

And it looks like there are a few user packages I installed that went there:

![image](https://user-images.githubusercontent.com/1360539/88319880-a4014800-ccd1-11ea-9530-0f6a094eaa23.png)

I do not truly know what is considered the equivalent of `/usr/local/bin` for Windows, unfortunately. Maybe there isn't one. Most self-installers I've seen add their install directory to the user PATH. Git for Windows, VSCode, Docker...

Maybe we need to look into providing a self-installer for Windows to resolve this issue, and recommend using `winget` or downloading the self-installer directly?

Those don't seem like action items we need to make here to address this issue.

Anecdotally though, my `%SystemRoot%\system32` does not appear to have any apps I installed recently. They all seem to be Windows DLLs or system executables like `print.exe` and `replace.exe`. If we infer that system32 is meant for OS executables, I do not think that is the correct location we should recommend to users to move their executables.

Did you check your system environment variables?

![image](https://user-images.githubusercontent.com/1360539/88320934-33f3c180-ccd3-11ea-97e7-dcb57f4cfac8.png)

Reply:
I do appear to have WindowsApps as part of my default user path, though I wonder if that's meant to be reserved for UWP apps from the Microsoft Store.

![image](https://user-images.githubusercontent.com/1360539/88321038-62719c80-ccd3-11ea-8ca3-da5c948f84ac.png)

Reply:
What I have done in the past has created a `bin/` directory in my `%USERPROFILE%`, then added that to my user environment variables.

https://superuser.com/questions/983138/what-is-the-equivalent-of-the-bin-directory-for-windows/983171 seems to have the same answer here too. There is no real equivalent here. It's a convention for Linux users, but there doesn't appear to be a standard convention for Windows.

Reply:
Yeah, so I think a windows installer might be in order here. My environment variables were quite bare. I only had 3 different items in my PATH, and one of those was from Cargo